README.md

```diff
@@ -42,20 +45,20 @@ module Checklists
     def human_name
       "Blarg!"
     end
-    
+
     def checklist
     '- [ ] Removed extraneous blargh'.strip
     end
-    
+
     def relevant_for(files)
-      files.select do |file| 
+      files.select do |file|
         file.ends_with(".rb") && IO.read(file) =~ /blargh/
       end
     end
   end
 
   module_function
-  
+
```

Comment:
Sorry for these whitespace changes. Didn't notice that, it's something my editor does by default. Happy to remove them if it's annoying!

Reply:
That's all good, I try to remove trailing whitespace too!